mattermost/mattermost-webapp
# [MM-23872] Fix signup callback sending user to undefined team name on error

#### Summary
- If the team name is undefined (i.e. the user is not being invited to a specific team) then dont append it to the URL 
- Adds some tests to ensure the behaviour 

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-23872


#### Test steps:

- Ensure that email verification is turned on for the server
- Ensure that the server is open for signups
- On login page, click Create and create a new account using email and password
- Observe the URL and ensure that there are no undefined url parameters
